src/utils/todo/task-filing/button-loader.ts

```diff
@@ -137,6 +139,7 @@ const showDeleteOnly = (todoItem: TodoItemI): boolean => {
       return false
     case FilingTypes.ALTERATION:
     case FilingTypes.DISSOLUTION:
+      return filingSubType === FilingSubTypeE.DISSOLUTION_ADMINISTRATIVE
```

Comment:
If you're in this code again, please consider leaving a comment explaining this.